### PR TITLE
Fix incorrect phonetics 著 words

### PR DESCRIPTION
**Pull request by xatier**
Ref:

https://dict.revised.moe.edu.tw/search.jsp?md=1&word=%E8%91%97#searchL https://www.moedict.tw/%E4%B8%89%E5%8D%81%E5%85%AD%E8%91%97%EF%BC%8C%E8%B5%B0%E7%82%BA%E4%B8%8A%E8%91%97 https://www.moedict.tw/%E8%91%97%E6%89%8B
https://www.moedict.tw/%E6%8B%99%E8%91%97
https://www.moedict.tw/%E6%B2%89%E8%91%97
https://www.moedict.tw/%E8%91%97%E5%AF%A6
https://www.moedict.tw/%E8%91%97%E6%80%A5
https://www.moedict.tw/%E9%A1%AF%E8%91%97

### CODE REVIEW

**Comment by xatier** [comment]
@lukhnos please take a look :pray:

File: Source/Data/BPMFMappings.txt [hunk 1]
```diff
@@ -75890,7 +75891,6 @@
 沈船 ㄔㄣˊ ㄔㄨㄢˊ
 沈落 ㄔㄣˊ ㄌㄨㄛˋ
 沈葆楨 ㄕㄣˇ ㄅㄠˇ ㄓㄣ
-沈著 ㄔㄣˊ ㄓㄨㄛˊ
```
Comment (lukhnos) [inline Source/Data/BPMFMappings.txt]: This seems to be the cause of the build failure: https://github.com/openvanilla/McBopomofo/actions/runs/9184669125/job/25257375803?pr=484

```
bin/cook.py PhraseFreq.txt BPMFMappings.txt BPMFBase.txt BPMFPunctuations.txt Symbols.txt Macros.txt data.txt
Error:  沈著 key mismatches.
```

**Review by lukhnos — CHANGES_REQUESTED** [review]

File: Source/Data/BPMFMappings.txt (hunk 1 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: Oops, thanks for pointing this out. I've removed that from `phrase.occ`. 

`沈著` is incorrect, it should be `沉著`

https://www.moedict.tw/%E6%B2%89%E8%91%97

File: Source/Data/BPMFBase.txt [hunk 2]
```diff
@@ -1863,7 +1863,6 @@
 貯 ㄓㄨˇ zhu3 5j3 big5
 渚 ㄓㄨˇ zhu3 5j3 big5
 矚 ㄓㄨˇ zhu3 5j3 big5
-著 ㄓㄨˇ zhu3 5j3 big5
```
Comment (lukhnos) [inline Source/Data/BPMFBase.txt]: 不好意思現在才想到：這個是原有 xcin bpmf.cin 收錄的讀音。雖然不是教育部字辭典收的音，但基於歷史因素，我們應該保留。

**Review by lukhnos — CHANGES_REQUESTED** [review]
Please roll back the changes to `BPMFBase.txt` and `phrase.occ` as well as the deletion of the entry for `沈著`, for the reason commented. Otherwise this LGTM. Thanks!

File: Source/Data/BPMFMappings.txt (hunk 1 above)
Comment (lukhnos) [reply Source/Data/BPMFMappings.txt]: 這個應該保留，因為詞庫收了同類詞：沉船/沈船、沉默/沈默等等。

File: Source/Data/phrase.occ [hunk 3]
```diff
@@ -77943,7 +77943,6 @@
 沈船 7
 沈落 4
 沈葆楨 15
-沈著 12
```
Comment (lukhnos) [inline Source/Data/phrase.occ]: 我認為這個應該 revert，請參見我在 Source/Data/BPMFMappings.txt 留的意見。

**Review by lukhnos — CHANGES_REQUESTED** [review]

File: Source/Data/BPMFMappings.txt (hunk 1 above)
Comment (xatier) [reply Source/Data/BPMFMappings.txt]: sure, let me revert this.

**Comment by xatier** [comment]
@lukhnos thanks for the feedbacks, I've reverted those deletions and squashed the commits so we have cleaner commit history.

**Review by lukhnos — APPROVED** [review]
Thank you!

File: Source/Data/BPMFMappings.txt (hunk 1 above)
Comment (tianjianjiang) [reply Source/Data/BPMFMappings.txt]: @xatier @lukhnos FYI: 而且「沉」一向很有爭議。以某些古籍和主管機關的看法來說，「沉」才是異體字。

Comment (xatier) [reply Source/Data/BPMFMappings.txt]: Interesting, my understanding is that `沈` is used mostly as a surname, and `沉` is used for `sunk`.

Comment (xatier) [reply Source/Data/BPMFMappings.txt]: https://dict.variants.moe.edu.tw/dictView.jsp?ID=23377

**Pull request merged**